packages/clerk-js/src/ui/contexts/utils.ts

```diff
@@ -13,3 +13,22 @@ export function assertContextExists(contextVal: unknown, providerName: string):
     clerkCoreErrorContextProviderNotFound(providerName);
   }
 }
+
+export function isRedirectForSSOFlow(clerk: Clerk, redirectUrl: string): boolean {
+  const fapiUrl = clerk.publishableKey || clerk.frontendApi;
```

Comment:
⛔ The publishableKey is the base64(`pk_(test|live)_${frontendApi}`) value. If not mistaken, `clerk.frontendApi` should be always available. It should be used to be compared with the `url.host`.

Reply:
Hmm the reason I had this logic in the first place is that `frontendApi` is marked deprecated here: https://github.com/clerkinc/javascript/blob/main/packages/types/src/clerk.ts#L62 - is that not the case? For now, I can add parsing for the publishable key, but we can remove it if `frontendApi` will always be available going forward.

Reply:
Gotcha, thanks for the clarification!